### DISCUSSION TITLE
Specify encoding of csv file exported from panels

### DISCUSSION THREAD

**Discussion by elijahxb** [grafana/grafana#37032] [Dashboards, Panels & Visualizations]
**I hope we can specify encoding of the csv file exported from a panel.**:

**We often open export data from panels on windows platform in china.
We have to open the csv files and change encoding from utf-8-BOM to ANSI/GB2312 manually， or we will get unintelligible texts.
Even worse, on some scenes, the unintelligible texts can cause separation character muddled.**:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
